spec_driven_model/models/spec_import.py

```diff
@@ -79,7 +79,10 @@ def _build_attr(self, node, fields, vals, path, attr):
         if (
             (
                 attr[1].type == str
-                or not any(["odoo.addons." in str(i) for i in attr[1].type.__args__])
+                or (
+                    not any(["odoo.addons." in str(i) for i in attr[1].type.__args__])
+                    and not any(["nfelib." in str(i) for i in attr[1].type.__args__])
```

Comment:
eu entendo porque vc precisou acrescentar esse teste, porem não ta legal depender de algo chamado nfelib neste modulo. Eu resolvi isso de forma robusta e comentei aqui: https://github.com/OCA/l10n-brazil/pull/2737#issuecomment-1774158629

Reply:
Resolvido. Assim que for feito o merge de #2737, eu faço o rebase deste PR